[2022-11-19T17:33:32.022+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data_v1.download_dataset_task manual__2022-11-19T17:31:11.536081+00:00 [queued]>
[2022-11-19T17:33:32.046+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: green_taxi_data_v1.download_dataset_task manual__2022-11-19T17:31:11.536081+00:00 [queued]>
[2022-11-19T17:33:32.047+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-19T17:33:32.047+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-11-19T17:33:32.047+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-19T17:33:32.080+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-11-19 17:31:11.536081+00:00
[2022-11-19T17:33:32.114+0000] {standard_task_runner.py:55} INFO - Started process 57503 to run task
[2022-11-19T17:33:32.127+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data_v1', 'download_dataset_task', 'manual__2022-11-19T17:31:11.536081+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_homework.py', '--cfg-path', '/tmp/tmpkwc26_fk']
[2022-11-19T17:33:32.134+0000] {standard_task_runner.py:83} INFO - Job 100: Subtask download_dataset_task
[2022-11-19T17:33:32.567+0000] {task_command.py:376} INFO - Running <TaskInstance: green_taxi_data_v1.download_dataset_task manual__2022-11-19T17:31:11.536081+00:00 [running]> on host b8836091f311
[2022-11-19T17:33:32.845+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-11-19T17:33:33.294+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=green_taxi_data_v1
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-19T17:31:11.536081+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-19T17:31:11.536081+00:00
[2022-11-19T17:33:33.296+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-19T17:33:33.321+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSLf https://d37ci6vzurychx.cloudfront.net/trip-data/green_tripdata_2022-11.parquet > /opt/***/green_tripdata_2022-11.parquet']
[2022-11-19T17:33:33.451+0000] {subprocess.py:86} INFO - Output:
[2022-11-19T17:33:34.047+0000] {subprocess.py:93} INFO - curl: (22) The requested URL returned error: 403
[2022-11-19T17:33:34.053+0000] {subprocess.py:97} INFO - Command exited with return code 22
[2022-11-19T17:33:34.107+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 22.
[2022-11-19T17:33:34.141+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=green_taxi_data_v1, task_id=download_dataset_task, execution_date=20221119T173111, start_date=20221119T173332, end_date=20221119T173334
[2022-11-19T17:33:34.216+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 100 for task download_dataset_task (Bash command failed. The command returned a non-zero exit code 22.; 57503)
[2022-11-19T17:33:34.310+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-11-19T17:33:34.392+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
